### PR TITLE
fix checking for python3-all (instead of python-all) when using only Python 3

### PR DESCRIPTION
**Pull request by Sispheor**
The "changes" can look scary. I formatted the full script to be pip compliant.
This fix the 

On Ubuntu 20.04 I needed to install dh-python package too.

Fixes #140 and #153.

### CODE REVIEW

**Comment by dirk-thomas** [comment]
Thank you for creating a PR for the changes proposed in #140 and to address the feedback from there.

Atm this PR contains multiple unrelated changes. Please focus on one change in a single PR and create separate PRs for unrelated changes. This allows each PR to be reviewed independently. In this case please don't change the version number (that will happen independently when a new release is being made by the maintainers). Also please don't include style-only changes in the same PR but move that into a separate PR. Thank you.

**Comment by Sispheor** [comment]
@dirk-thomas PR updated

File: stdeb/util.py [hunk 1]
```diff
@@ -1413,18 +1413,28 @@ def build_dsc(debinfo,
...
+                             'that use python-support.' % (
+                                 PYTHON_ALL_MIN_VERS,))
+
+        if debinfo.with_python3:
+            python3_defaults_version_str = get_version_str('python3-all')
+
+            if len(python3_defaults_version_str) == 0:
+                log.warn('This version of stdeb requires python3-all >= %s, '
+                         'but you do not have this package installed. '
+                         'Could not check compatibility.' % PYTHON_ALL_MIN_VERS)
```
Comment (dirk-thomas) [inline stdeb/util.py:1437]: Showing the Python 2 minimum version here seem wrong?

Comment (dirk-thomas) [reply stdeb/util.py:1437]: @Sispheor Friendly ping.

Comment (Sispheor) [reply stdeb/util.py:1423]: What do you mean?

Comment (Sispheor) [reply stdeb/util.py:1423]: The current [setup](https://github.com/astraw/stdeb/blob/master/setup.py#L30) point to python 3 with no specific minor version.
The var PYTHON_ALL_MIN_VERS is set to '2.6.6-3'.
You want me to add a new var called PYTHON3_ALL_MIN_VERS = 3 ?

**Comment by Sispheor** [comment]
I've updated the PR. Hope it's ok. I selected the last version of python3-all from the last Ubuntu LTS as a reference.

File: stdeb/util.py (hunk 1 above)
Comment (dirk-thomas) [reply stdeb/util.py]: The previous message said something like "You need python3-all > 2.6.x" which doesn't make any sense.

What motivates to pick a specific `PYTHON3_ALL_MIN_VERS`? Is there any specific constrain or can the message just refer to `python3-all` without a minimum version?

**Comment by dirk-thomas** [comment]
(Please avoid force pushing changes since every time the whole patch needs to be re-reviewed with no indication what changes compared to the previous iteration. The PR can be merged with squashing so only a single commit appears afterwards on the target branch.)

File: stdeb/util.py (hunk 1 above)
Comment (dirk-thomas) [reply stdeb/util.py]: @Sispheor Friendly ping.

Comment (dirk-thomas) [reply stdeb/util.py]: In order to move forward on this I have removed the `PYTHON3_ALL_MIN_VERS` in 05734f61f6ca1e0ba1bf0844a16daf7ec2740413.

**Comment by dirk-thomas** [comment]
Thanks for the contributions.

I have updated the patch based on my previous feedback in 05734f61f6ca1e0ba1bf0844a16daf7ec2740413 and c76d7a878ba928f00842cfb714bcf0039415f606.

**Pull request merged**